x-pack/plugins/maps/public/connected_components/map/mb/tooltip_control/tooltip_control.js

```diff
@@ -127,9 +131,10 @@ export class TooltipControl extends React.Component {
     }
 
     const popupAnchorLocation = justifyAnchorLocation(e.lngLat, targetMbFeature);
-    const features = this._getIdsForFeatures(mbFeatures);
+    const featuresAndMeta = this._getIdsAndFeaturesMeta(mbFeatures);
     this.props.openOnHoverTooltip({
-      features: features,
+      mbFeatures,
```

Comment:
Signature for `openOnHoverTooltip` is `export function openOnHoverTooltip(tooltipState: TooltipState)`. `TooltipState` does not contain `mbFeatures`